src/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.Renamed.cs

```diff
@@ -150,7 +150,7 @@ public static void FileSystemWatcher_Moved_NestedDirectoryRootWithoutSubdirector
 
                 using (var dir2 = new TemporaryTestDirectory(Path.Combine(dir1.Path, "dir2")))
                 {
-                    Utility.ExpectNoEvent(createdOccured, "dir2 created", Utility.WaitForCreationTimeoutInMs);
+                    Utility.ExpectNoEvent(createdOccured, "dir2 created");
```

Comment:
Hmm it might not need 30 seconds but the default 500ms is too slow; I saw the Linux implementation miss actual creation events at 500ms so we could get false negatives leaving it at 500ms